docs/developer/eventing/sources/kafka-source/README.md

```diff
@@ -13,63 +6,80 @@ Tutorial on how to build and deploy a `KafkaSource` event source.
 
 The `KafkaSource` reads all the messages, from all partitions, and sends those messages as CloudEvents via HTTP to its configured `sink`.
 
-*NOTE:* In case you need a more sophisticated Kafka Consumer, with direct access to specific partitions or offsets you might want to implement a _Kafka Consumer_, using one of the available Apache Kafka SDKs, to handle the messages yourself, rather than using the Knative `KafkaSource`.
+!!! note
+    If need a more sophisticated Kafka Consumer, with direct access to specific partitions or offsets, you can implement a Kafka Consumer by using one of the available Apache Kafka SDKs.
 
-## Prerequisites
+<!--TODO: Check if this note is out of scope; should we not mention anything beyond the direct Knative/Kafka integration we provide?-->
 
-- A Kubernetes cluster with [Knative Kafka Source installed](../../../admin/install/).
+## Installing the Kafka event source CRD
 
-## Apache Kafka Topic (Optional)
+Install the Kafka event source by running the command:
 
-1. If using Strimzi, you can set a topic modifying `source/kafka-topic.yaml`
-   with your desired:
+```bash
+kubectl apply -f https://storage.googleapis.com/knative-nightly/eventing-kafka/latest/source.yaml
+```
+<!--Taken from https://github.com/knative-sandbox/eventing-kafka - not sure if there's a newer command-->
 
-- Topic
-- Cluster Name
-- Partitions
-- Replicas
+## Optional: Create a Kafka topic
 
-  ```yaml
-  apiVersion: kafka.strimzi.io/v1beta2
-  kind: KafkaTopic
-  metadata:
-    name: knative-demo-topic
-    namespace: kafka
-    labels:
-      strimzi.io/cluster: my-cluster
-  spec:
-    partitions: 3
-    replicas: 1
-    config:
-      retention.ms: 7200000
-      segment.bytes: 1073741824
-  ```
-
-2. Deploy the `KafkaTopic`
-
-   ```bash
-   $ kubectl apply -f strimzi-topic.yaml
-   kafkatopic.kafka.strimzi.io/knative-demo-topic created
-   ```
+1. If you are using Strimzi, create a KafkaTopic:
 
-3. Ensure the `KafkaTopic` is running.
+    ```yaml
+    apiVersion: kafka.strimzi.io/v1beta2
+    kind: KafkaTopic
+    metadata:
+      name: knative-demo-topic
+      namespace: kafka
+      labels:
+        strimzi.io/cluster: my-cluster
+    spec:
+      partitions: 3
+      replicas: 1
+      config:
+        retention.ms: 7200000
+        segment.bytes: 1073741824
+    ```
 
-   ```bash
-   $ kubectl -n kafka get kafkatopics.kafka.strimzi.io
-   NAME                 AGE
-   knative-demo-topic   16s
-   ```
+2. Deploy the `KafkaTopic` by running the command:
```

Comment:
I think 1) and 2) could be merged. For example, "`kubectl apply` the following manifest to create a Kafka topic via Strimzi."

Reply:
I think as part of our docs style guide we want to keep these sort of steps separate, e.g. 1. create the file, 2. apply it. We were combining it before with `kubectl apply -f <<EOF` type commands but there were issues with it not working on Windows so we decided to keep it separated.